### PR TITLE
partially implemented paddle-quantile function

### PR DESCRIPTION
**Pull request by elabongaatuo**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_stat.py [hunk 1]
```diff
@@ -128,3 +128,41 @@ def test_paddle_median(
...
+    q=st.floats(0.0, 1.0),
+)
+def test_paddle_quantile(
+    *,
+    q,
+    dtype_x_and_axis,
+    keepdim,
+    on_device,
+    fn_tree,
+    frontend,
```
Comment (paulaehab) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_stat.py:154]: Please add  backend_fw, here to detect the backend for tests.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_stat.py [hunk 2]
```diff
@@ -128,3 +128,41 @@ def test_paddle_median(
...
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        x=x[0],
+        q=q,
+        axis=axis,
+        keepdim=keepdim,
```
Comment (paulaehab) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_stat.py:167]: Please pass  backend_to_test=backend_fw, in test_frontend_function function

**Review by paulaehab — COMMENTED** [review]
Please open sub-task issue for this function and put it in the comments refer to this guide here
(https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues)

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**